src/parser/parser.ts

```diff
@@ -185,16 +191,28 @@ export function parseStudyConfig(fileData: string): ParsedStudyConfig {
     validatedData = false;
   }
 
-  let errors: Required<ParsedStudyConfig>['errors'] = [];
-  let warnings: Required<ParsedStudyConfig>['warnings'] = [];
+  let errors: Required<ParsedConfig<StudyConfig>>['errors'] = studyValidate.errors || [];
+  let warnings: Required<ParsedConfig<StudyConfig>>['warnings'] = [];
 
   // We can only run our custom validator if the schema validation passes
   if (validatedData) {
-    const { errors: parserErrors, warnings: parserWarnings } = verifyStudyConfig(data!);
-    errors = [...(studyValidate.errors || []), ...parserErrors];
-    warnings = parserWarnings;
+    const importedLibraries = data!.importedLibraries || [];
```

Comment:
Is there some way for us to avoid having to force (!) the data object, some type check above?

Reply:
Fixed in 60865a3